demisto_sdk/commands/common/update_id_set.py

```diff
@@ -418,6 +418,43 @@ def get_incident_field_data(path, incidents_types_list):
     return {id_: data}
 
 
+def get_indicator_type_data(path):
+    data = OrderedDict()
+    json_data = get_json(path)
+
+    id_ = json_data.get('id')
+    name = json_data.get('details', '')
+    fromversion = json_data.get('fromVersion')
+    toversion = json_data.get('toVersion')
+    associated_integration = json_data.get('reputationCommand')
```

Comment:
is it really an integration rather than a command?

Reply:
Yes, its either `!ip/!email`,etc...